api/net/BackgroundWorkItem/BackgroundWorkerQueue.cs

```diff
@@ -2,37 +2,20 @@
 
 namespace TNO.API.BackgroundWorkItem;
 
-/// <summary>
-///
-/// </summary>
-public interface IBackgroundTaskQueue
-{
-    /// <summary>
-    ///
-    /// </summary>
-    /// <param name="cancellationToken"></param>
-    /// <returns></returns>
-    Task<Func<CancellationToken, Task>> DequeueAsync(CancellationToken cancellationToken);
-    /// <summary>
-    ///
-    /// </summary>
-    /// <param name="workItem"></param>
-    void QueueBackgroundWorkItem(Func<CancellationToken, Task> workItem);
-}
 /// <summary>
 ///
 /// </summary>
 public class BackgroundTaskQueue : IBackgroundTaskQueue
 {
-    private ConcurrentQueue<Func<CancellationToken, Task>> _workItems = new ConcurrentQueue<Func<CancellationToken, Task>>();
-    private SemaphoreSlim _signal = new SemaphoreSlim(0);
+    private readonly ConcurrentQueue<Func<CancellationToken, Task>> _workItems = new();
```

Comment:
@kylgarmor you should add a linter to your vscode.  It will also catch many of the nullable issues.